pkg/openSearch/openSearchClient/error.go

```diff
@@ -69,16 +53,6 @@ func (o *OpenSearchResourceNotFound) Error() string {
 	return o.Message
 }
 
-func NewOpenSearchResourceNotFound(message string) *OpenSearchResourceNotFound {
-	return &OpenSearchResourceNotFound{
-		Message: message,
-	}
-}
-
```

Comment:
Why has this been removed?

Reply:
The functions described in the PR notes do not have any references, hence the removal. Let me know if you want me to restore them.

Reply:
I would restore them because all other error types have a constructor

Reply:
Fixed in c52072a0a084102ada43709c85b442b29bcf5ab2